blocks/identity-block/themes/news.json

```diff
@@ -685,6 +771,64 @@
 			"desktop": {}
 		}
 	},
+	"social-sign-on-button-container-google": {
```

Comment:
@LauraPinilla Can you please update https://github.com/WPMedia/arc-themes-blocks/blob/ASUB-8521-SocialMediaButtons/.storybook/themes/news.scss as well to match changes in this file.